dags/pytorch_xla/nightly.py

```diff
@@ -47,7 +47,7 @@
 )
 
 
-@task_group
+@task_group(prefix_group_id=False)
```

Comment:
What does `prefix_group_id=False` do?

Reply:
It changes the actual task name rather than just grouping the tasks. `post_process` is expecting `pt-nightly-llama2-pjrt-infer-func-v4-8-1vm-1vm.provision.create_queued_resource.wait_for_ready_queued_resource`. With `prefix_group_id=True` (default) this becomes `llama. pt-nightly-llama2-pjrt-infer-func-v4-8-1vm-1vm.provision.create_queued_resource.wait_for_ready_queued_resource`

Reply:
That means with `prefix_group_id=True`, the task name as an extra `llama. ` at the beginning?

Reply:
Right, it prepends the name of the `@task_group`ed function